packages/plugins/dev.ts

```diff
@@ -57,7 +57,7 @@ const deployConfigForceCommit = async (fileName: string, silent: boolean) => {
   const { bundleId } = await hre.run('chugsplash-commit', {
     configPath,
     ipfsUrl: '',
-    commitToIpfs: false,
+    commitToIpfs: true,
```

Comment:
I found the dev script did not actually commit to IPFS until I flipped this to true.